Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-findOneAndUpdate.txt

MongoDB\\Collection::findOneAndUpdate()
=======================================

.. meta::

>    :description: Discover how to use `findOneAndUpdate()` in PHP to find and update a single document in a MongoDB collection, with options for projection, sorting, and more.

Reviewer: ```suggestion
   :description: Find and update a single document in a collection with the MongoDB PHP Library, with options for projection, sorting, and more.
```
